include/nccl_ofi_rdma.h

```diff
@@ -474,6 +501,16 @@ typedef struct nccl_net_ofi_rdma_send_comm {
 	nvtxDomainHandle_t nvtx_domain[NCCL_OFI_N_NVTX_DOMAIN_PER_COMM];
 #endif
 
+	nccl_ofi_deque_elem_t cleanup_list_elem;
+
+	pthread_mutex_t receive_close_lock;
+	bool received_close_message;
```

Comment:
Between comm_active and this, we might as well keep `state` as an enum.

Reply:
I don't think that works, because we can receive the close message before or after `send_close` is called. So there are four possible states: `comm_active` and `received_close_message` can independently be true or false. This is not easily expressed as an enum.